### ISSUE TITLE
Keyboard navigation using keypress does not work in all browsers

### ISSUE THREAD

**Issue by tastypopsicle** [Codeinwp/Nivo-Slider-jQuery#190]
Keypress is not supported in Webkit browsers. The use of keypress in v2.5.2 line 188 should be changed to keyup for cross browser compatibility.
